hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/infoschema/MappingsTable.java

```diff
@@ -37,12 +37,12 @@
     private static final String NAME = "mappings";
 
     private static final List<TableField> FIELDS = asList(
-            new TableField("mapping_catalog", QueryDataType.VARCHAR, false),
-            new TableField("mapping_schema", QueryDataType.VARCHAR, false),
-            new TableField("mapping_name", QueryDataType.VARCHAR, false),
-            new TableField("mapping_external_name", QueryDataType.VARCHAR, false),
-            new TableField("mapping_type", QueryDataType.VARCHAR, false),
-            new TableField("mapping_options", QueryDataType.VARCHAR, false)
+            new TableField("table_catalog", QueryDataType.VARCHAR, false),
+            new TableField("table_schema", QueryDataType.VARCHAR, false),
+            new TableField("table_name", QueryDataType.VARCHAR, false),
+            new TableField("table_external_name", QueryDataType.VARCHAR, false),
+            new TableField("table_type", QueryDataType.VARCHAR, false),
+            new TableField("table_options", QueryDataType.VARCHAR, false)
```

Comment:
I'd rename just the first three columns. These are the primary key used from the `columns` view. The last three are custom, I'll keep the word `mapping` in those.

Reply:
Would not it be confusing? There is also `table_type` in the standard.

Reply:
`table_type` in the standard is something else: 
![image](https://user-images.githubusercontent.com/9109877/127449811-ca6d28dd-3a9f-4c1d-abcc-49a13f8be267.png)
For mappings it's the connector type.

There is a style in the relational data design that columns containing the same data should be named the same and columns containing different data must be named differently. This is supported by the `USING` keyword in joins: `SELECT * FROM mappings JOIN columns USING(table_catalog, table_schema, table_name)`. That's why the mapping name should be in `table_name` column (even though it looks illogical). That's also why the view name in `views` is in `table_name` column. That's also why `mapping_type` shouldn't be renamed to `table_type`, because it's already used for something else.